tests/acceptance-test/src/test/java/com/quorum/tessera/test/rest/VersionIT.java

```diff
@@ -25,51 +25,51 @@
+                .collect(Collectors.toList());
 
         allUris.forEach(
-                u -> {
-                    final String version = client.target(u).path("/version").request().get(String.class);
-                    assertThat(version).isEqualTo(Version.getVersion());
-                });
+            u -> {
+                final String version = client.target(u).path("/version").request().get(String.class);
+                assertThat(version).isEqualTo(Version.getVersion());
+            });
     }
 
     @Test
     public void getDistributionVersion() {
         final List<URI> allUris =
-                partyHelper
-                        .getParties()
-                        .flatMap(p -> Stream.of(p.getQ2TUri(), p.getP2PUri()))
-                        .collect(Collectors.toList());
+            partyHelper
+                .getParties()
+                .flatMap(p -> Stream.of(p.getQ2TUri(), p.getP2PUri()))
+                .collect(Collectors.toList());
 
         allUris.forEach(
-                u -> {
-                    final String version = client.target(u).path("/version/distribution").request().get(String.class);
-                    assertThat(version).isEqualTo(Version.getVersion());
-                });
+            u -> {
+                final String version = client.target(u).path("/version/distribution").request().get(String.class);
+                assertThat(version).isEqualTo(Version.getVersion());
+            });
     }
 
     @Test
     public void getSupportedVersions() {
 
         List<URI> allUris =
-                partyHelper
-                        .getParties()
-                        .flatMap(p -> Stream.of(p.getQ2TUri(), p.getP2PUri()))
-                        .collect(Collectors.toList());
+            partyHelper
+                .getParties()
+                .flatMap(p -> Stream.of(p.getQ2TUri(), p.getP2PUri()))
+                .collect(Collectors.toList());
 
         allUris.forEach(
-                u -> {
-                    JsonArray versions = client.target(u).path("version").path("api").request().get(JsonArray.class);
-                    assertThat(
-                                    versions.stream()
-                                            .map(JsonString.class::cast)
-                                            .map(JsonString::getString)
-                                            .toArray(String[]::new))
-                            .containsExactly("1.0", "2.0", "2.1");
-                });
+            u -> {
+                JsonArray versions = client.target(u).path("version").path("api").request().get(JsonArray.class);
+                assertThat(
+                    versions.stream()
+                        .map(JsonString.class::cast)
+                        .map(JsonString::getString)
+                        .toArray(String[]::new))
+                    .containsExactly("1.0", "2.0", "2.1", "3.0");
```

Comment:
is this the only actual change, the rest is whitespace?

Reply:
yes